src/main/java/com/users/service/UserService.java

```diff
@@ -48,10 +53,21 @@ public UserDto updateUser(Long id, UserDto userDto) {
         return userMapper.toDto(savedUser);
     }
 
-    public List<UserDto> findAllUsers() {
+    public Page<UserDto> findAllUsers(UserCriteria criteria) {
+        log.debug("Request to get all users");
+        Specification<User> searchSpecification = new UserSpecification(criteria);
+        if (criteria.getPageRequestDto() == null) {
+            criteria.setPageRequestDto(new PageRequestDto());
+        }
+        Pageable pageable = criteria.getPageRequestDto().getPageable(criteria.getPageRequestDto());
```

Comment:
cred ca ai putea pune direct in controller pageable

Reply:
am zis sa nu pun in controller cod prea mult ca asa imi spusesei

Reply:
ma refer ca si parametru, sa nu ai PageRequestDTO sa ai Pageable direct public ResponseEntity<Page<UserDto>> findAll(UserCriteriaEasy criteria, Pageable pageable) {

Reply:
am facut asa, am adaptat tot si am sters PageRequestDto